### PR TITLE
Only copy URL to clipboard when !open

### PR DESCRIPTION
**Pull request by billyjanitsch**
This PR contains a:

- [ ] **bugfix**
- [x] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [ ] **metadata update**

### Motivation / Use-Case

This patch is a partial solution for #33: it disables the clipboard feature if the user has configured the `open` option (which clearly makes the clipboard feature redundant).

It _doesn't_ introduce a way to configure the clipboard feature or otherwise change the default, pending more discussion in #33.

### CODE REVIEW

**Comment by billyjanitsch** [comment]
CI wants me to add a test. Fair enough :)

**Comment by billyjanitsch** [comment]
@shellscape how would you prefer me to test a config option like this? By adding a new config fixture to `test/fixtures/`? By modifying the basic one in an individual test?

**Comment by shellscape** [comment]
Should be able to sneak that into this test, just checking that the clipboard is empty: https://github.com/webpack-contrib/webpack-serve/blob/master/test/tests/options.js#L135

And you can modify the `then` like so to double check the options was mutated properly:

```js
serve({ config }).then(({ close, options }) => {
```

**Comment by shellscape** [comment]
@billyjanitsch I just merged a PR that makes the clipboard stuff an option, so you'll probably need to rebase for that. We should also add a note to the README within the `open` option block that states that the clipboard copying is disabled when using the `open` option.

**Comment by billyjanitsch** [comment]
Awesome! Sorry for the delay on this PR, I'll finish it up today.

**Comment by billyjanitsch** [comment]
@shellscape let me know how that looks -- happy to make further changes

File: README.md [hunk 1]
```diff
@@ -284,6 +284,8 @@ that matches:
 }
 ```
 
+This option disables the `clipboard` option.
```
Comment (shellscape) [inline README.md]: I'd change this to:

```md
_Note: Using the `open` option will disable the `clipboard` option.
```

File: test/tests/options.js [hunk 2]
```diff
@@ -149,9 +149,11 @@ describe('webpack-serve Options', () => {
   t('should accept an open:Boolean option', (done) => {
     const config = load('./fixtures/basic/webpack.config.js');
     config.serve.open = true;
+    clip.writeSync('foo');
 
     serve({ config }).then(({ close }) => {
       hook = (...args) => {
+        assert.equal(clip.readSync(), 'foo');
```
Comment (shellscape) [inline test/tests/options.js]: looks good 👍 

can you add a comment above this line to note that it tests that the clipboard option is disabled when open is used?

**Comment by billyjanitsch** [comment]
@shellscape done, let me know if there's anything else!

**Pull request merged**